styles.css

```diff
@@ -1,12 +1,34 @@
 /*Maak hier jouw CSS!*/
 
+.container {
+    display: grid;
+    grid-gap: 16px 16px;
+    background-color: rgb(252, 238, 240);
+    color: white;
+    padding: 15px;
```

Comment:
deze padding was hier niet nodig omdat je al grid-gap hebt gebruikt (die is overrulend blijkbaar)